rescript/plugin_setup.py

```diff
@@ -1141,6 +1144,8 @@
     parameter_descriptions={
                 'threads': VSEARCH_PARAM_DESCRIPTIONS['threads'],
                 'perc_identity': VSEARCH_PARAM_DESCRIPTIONS['perc_identity'],
+                'target_coverage': 'Minimum coverage of reference segment '
```

Comment:
This param description is a little confusing. Could you please make it clearer?

Also since this is `target_coverage`, shouldn't this be the minimum coverage of the target, which is the reference sequence segments in this case? As written now it sounds like this is coverage of the input sequences.

Reply:
I've updated the help text to be more clear.